### PR TITLE
Run 3 background cards for WWJTo2L2Nu_NNLOPS_TuneCP5_13TeV-powheg-pyt…

### PR DESCRIPTION
**Pull request by srudra3**
Added Run 3  cards for background samples for HWW (WWJTo2L2Nu_NNLOPS_TuneCP5_13TeV-powheg-pyhia8) and HGG (GJet_Pt-20toInf_DoubleEMEnriched_MGG-40to80_TuneCP5_13TeV_Pythia8) 

1. WWJTo2L2Nu_NNLOPS_TuneCP5_13TeV-powheg-pyhia8: Modified the comEnergy and PDF
2. GJet_Pt-20toInf_DoubleEMEnriched_MGG-40to80_TuneCP5_13TeV_Pythia8 : Modified the comEnergy (Not sure if/how the cross-section needs to be updated)

Please verify if anything needs to be changed

### CODE REVIEW

File: bin/Powheg/production/Run3/13p6TeV/Higgs/WWJTo2L2Nu_NNLOPS/WWJTo2L2Nu_NNLOPS_13TeV.input [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# other settings (mostly for debugging purposes)
+#flg_debug 1        ! store extra event info for debugging
+#storemintupb 1    ! (default 0) if 1 save calls in a file
+
+lhrwgt_id 'st4'
+lhrwgt_descr 'Initial st4 event generation'
+
+# other parameters
+#wmasslow 10d0
+#wmasshigh 500d0
```
Comment (menglu21) [inline bin/Powheg/production/Run3/13p6TeV/Higgs/WWJTo2L2Nu_NNLOPS/WWJTo2L2Nu_NNLOPS_13TeV.input:55]: please modify the name to "**_13p6TeV.input"

File: genfragments/ThirteenPointSixTeV/GJet_Pt-20toInf_DoubleEMEnriched_MGG-40to80_TuneCP5_13TeV_Pythia8_cfi.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
```
Comment (menglu21) [inline genfragments/ThirteenPointSixTeV/GJet_Pt-20toInf_DoubleEMEnriched_MGG-40to80_TuneCP5_13TeV_Pythia8_cfi.py:3]: from Configuration.Generator.MCTunesRun3ECM13p6TeV.PythiaCP5Settings_cfi import *

**Review by menglu21 — COMMENTED** [review]
the file name should be "***13p6TeV***"

**Comment by srudra3** [comment]
Fixed the file names and the tune setting

**Comment by menglu21** [comment]
look fine to me now, @Saptaparna @mlizzo @Dominic-Stafford please have a check

**Pull request merged**